deploy/charts/operator/values.yaml

```diff
@@ -178,3 +178,20 @@ operator:
 
   # -- Affinity settings for the operator pod
   affinity: {}
+
+# -- All values for the registry API deployment and associated resources
+registryAPI:
+  # -- Container image for the registry API
+  image: "ghcr.io/stacklok/toolhive/thv-registry-api:latest"
+  # -- Service account configuration for the registry API
+  serviceAccount:
+    # -- Specifies whether a service account should be created
+    create: true
```

Comment:
what's the alternative, if the SA is not created?

Reply:
I just reused the pattern from `deploy/charts/operator/templates/serviceaccount.yaml` . 

I will get rid of this condition and just rely on the experimental feature one.